### ISSUE TITLE
Will Bumblebee be needed in Wayland? Are there plans to make Wayland more flexible about Optimus-like and multi-GPU rendering?

### ISSUE THREAD

**Issue by timofonic** [Bumblebee-Project/Bumblebee#746]
Hello.

Thanks for the project.

I see few activity, I hope the project gains traction again and the efforts gets mainlines asd much as possible.

Are there plans to collaborate with Wayland to make Bumblebee part of itself? In order to not require external dependencies, but make it aware of Optimus and multi-GPU rendering etc.

It would be really amazing if some Wayland devs are able to make the SPICE protocol to happen and use some smart way to even do local 3D rendering (OpenGL/Vulkan).

Kind regards.

**Comment by Lekensteyn** [comment]
AFAIK, Waylands design should remove the need for something like Bumblebee. Even with Xorg, PRIME is a good alternative (minus driver bugs).

**Comment by ArchangeGabriel** [comment]
Yes indeed, Wayland should be a step above what PRIME achieves. But currently and AFAIK, it does not support this (one of the reason Fedora is reluctant to switch to Wayland as default).
